2/src/Stock.java

```diff
@@ -1,5 +1,126 @@
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
 public class Stock {
     public static void main(String[] args) {
         System.out.println("Stock Management System");
+        Scanner sc = new Scanner(System.in);
+        Store items = new Store();
+        while (true){
+            System.out.println("어떤기능을 이용하실건가요?");
+            System.out.println("1. 물품 신청");
+            System.out.println("2. 물품 조회");
+            System.out.println("3. 종료");
+            String inp = sc.nextLine();
+            if (inp=="1") {
+                if (items.getSize()==5) {
+                    System.out.println("물품 신청은 최대 5개까지 가능합니다.");
+                    continue;
+                }
+
+                System.out.println("어떤 물품을 신청하시겠습니까?");
+                System.out.print("신청자: ");
+                String name = sc.nextLine();
+
+                System.out.print("물품명: ");
+                String item = sc.nextLine();
+
+                Integer quantity = input(sc, "수량");
+
+                Integer price = input(sc, "가격");
+
+                String link;
+                do {
+                    System.out.print("구매링크: ");
+                    link = sc.nextLine();
+                } while (!Checker.check(link));
+                Item i = new Item(name, item, quantity, price, link);
+                items.saveItem(i);
+                System.out.println("성공적으로 신청되었습니다.");
+            } else if (inp=="2") {
```

Comment:
자바에서 문자열 비교는 .equals를 사용합니다.
```suggestion
            } else if (inp.equals("2")) {
```